lib/active_interaction/filters/hash_filter.rb

```diff
@@ -51,13 +51,13 @@ def method_missing(*args, &block)
 
     private
 
-    def clean_value(h, name, filter, value)
-      h[name] = filter.clean(value[name])
+    def clean_value(h, name, filter, value, interaction)
+      h[name] = filter.clean(value[name], interaction)
     rescue InvalidValueError, MissingValueError
       raise InvalidNestedValueError.new(name, value[name])
     end
 
-    def raw_default
+    def raw_default(interaction)
```

Comment:
Should we just `*` this since we don't care about the params at all. They're just going through to the `super` call.